packages/api/src/utils/parser.ts

```diff
@@ -118,12 +118,20 @@ const parseOriginalContent = (document: Document): PageType => {
         return PageType.Profile
       case 'website':
         return PageType.Website
+      case 'tweet':
```

Comment:
Are these defined by opengraph? (Don't think it matters if they aren't, it will just default to unknown and then the user can set)

Reply:
No, I set these in the content_handlers @jacksonh

Reply:
ohhh smart. Nice work!